traffic_portal/app/src/common/modules/table/serverDeliveryServices/TableServerDeliveryServicesController.js

```diff
@@ -31,6 +33,32 @@ var TableServerDeliveryServicesController = function(server, deliveryServices, $
 			);
 	};
 
+	var confirmRemoveDS = function(ds, $event) {
+		if ($event) {
+			$event.stopPropagation(); // this kills the click event so it doesn't trigger anything else
+		}
+
+		var params = {
```

Comment:
`const` ?

Reply:
i guess i just moved this function around. i made no changes otherwise so would rather leave it as is and maybe (hopefully) some day circle back and use lets/const...